internal/backends/winit/glwindow.rs

```diff
@@ -368,24 +368,30 @@ impl<Renderer: WinitCompatibleRenderer + 'static> WindowAdapterSealed for GLWind
 
                 let sf = self.window.scale_factor();
 
-                winit_window.set_resizable(true);
-                winit_window.set_min_inner_size(if min_width > 0. || min_height > 0. {
+                let min_inner_size = if min_width > 0. || min_height > 0. {
                     Some(winit::dpi::PhysicalSize::new(min_width * sf, min_height * sf))
                 } else {
                     None
-                });
-                winit_window.set_max_inner_size(
-                    if max_width < i32::MAX as f32 || max_height < i32::MAX as f32 {
-                        Some(winit::dpi::PhysicalSize::new(
-                            (max_width * sf).min(65535.),
-                            (max_height * sf).min(65535.),
-                        ))
+                };
+                let max_inner_size = if max_width < i32::MAX as f32 || max_height < i32::MAX as f32
+                {
+                    Some(winit::dpi::PhysicalSize::new(
```

Comment:
(was present before, but is there a reason we convert to PhysicalSize instead of using the winit::dpi::LogicalSize , here and for the min size as well)

Reply:
I think that if we use the LogicalSize the scale factor override won't work.